components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -115,7 +115,9 @@ public int createApplication(ServiceProvider serviceProvider, String tenantDomai
             List<ApplicationMgtListener> listeners = ApplicationMgtListenerServiceComponent.getListners();
```

Comment:
Let's change return type of createApplication to void. The int value returned is the auto-generated table ID which should not be returned, and isn't used anywhere in the client side as well.